src/services/blockchain/ethereum/ethereum-blockchain-service.ts

```diff
@@ -131,7 +136,7 @@ export default class EthereumBlockchainService implements BlockchainService {
           throw new Error("Chain ID of connected blockchain changed from " + this.chainId + " to " + caip2ChainId)
         }
 
-        const txReceipt: providers.TransactionReceipt = await this.provider.waitForTransaction(txResponse.hash);
+        const txReceipt: providers.TransactionReceipt = await this.provider.waitForTransaction(txResponse.hash, 1, TRANSACTION_TIMEOUT);
```

Comment:
I guess TRANSACTION_TIMEOUT should be scaled with how frequently we run the anchor service. On mainnet I think it would be fine if this is up to 6h.

Reply:
I'll make it configurable